libbeat/kibana/transform_test.go

```diff
@@ -189,9 +189,20 @@ func TestTransformMisc(t *testing.T) {
 		{commonField: common.Field{}, expected: true, attr: "doc_values"},
 		{commonField: common.Field{DocValues: &truthy}, expected: true, attr: "doc_values"},
 		{commonField: common.Field{DocValues: &falsy}, expected: true, attr: "doc_values"},
+		{commonField: common.Field{Script: "doc[]"}, expected: false, attr: "doc_values"},
+		{commonField: common.Field{DocValues: &truthy, Script: "doc[]"}, expected: false, attr: "doc_values"},
 
-		// scripted always set to false
+		// script, scripted
 		{commonField: common.Field{}, expected: false, attr: "scripted"},
+		{commonField: common.Field{}, expected: nil, attr: "script"},
+		{commonField: common.Field{Script: "doc[]"}, expected: true, attr: "scripted"},
+		{commonField: common.Field{Script: "doc[]"}, expected: "doc[]", attr: "script"},
+
+		// language
+		{commonField: common.Field{}, expected: nil, attr: "lang"},
+		{commonField: common.Field{Language: "lucene"}, expected: nil, attr: "lang"},
+		{commonField: common.Field{Language: "lucene", Script: "doc[]"}, expected: "lucene", attr: "lang"},
+		{commonField: common.Field{Script: "doc[]"}, expected: "painless", attr: "lang"},
```

Comment:
@simianhacker I saw that in the index-pattern you used in `apm-server` the `readFromDocValues`  was set to `false` for scripted fields. Thus I added the logic here to set the field to false. Does this make sense?

You used `readFromDocValues` rather than `doc_values` - which one should we use for the Kibana index pattern?